docs/user-guide/manage-branches.md

```diff
@@ -2,6 +2,18 @@
 
 This page describes how to manage branches in the created component, whether it is an application, library, autotest or infrastructure.
 
+## Custom Pipeline Management
+
+There are two approaches to add an application with custom Tekton pipelines:
+
+  * **Using custom build tool and framework**: This approach involves [adding your application](../use-cases/tekton-custom-pipelines.md) with custom language type and framework.
+  * **Using add-ons**: This newer approach involves creating a component with custom Tekton pipelines in your [add-ons repository](../operator-guide/add-ons-overview.md) for each branch specifically and selecting it in the codebase branch settings.
+
+Unlike the [previous approach](../use-cases/tekton-custom-pipelines.md), the new one offers two main advantages:
+
+  * **Simplified management**: The new approach eliminates the need for strict naming conventions for resources and uses Argo CD for deploying Helm charts instead of pure Helm.
+  * **Branch-based logic**: This approach allows for the implementation of custom build and review pipeline logic based on the branch name of the codebase.
```

Comment:
we don't use the branch name as a flag here, we can just fart pipelines opposite the branch